.github/workflows/continous-integration.yml

```diff
@@ -537,6 +537,92 @@ jobs:
         run: |
           make test-integration
```

Comment:
The same make command is being used for `sequential` and `non-sequential integration` tests, does that mean we are running the same set of tests twice in the workflow?

Reply:
The `make test-integration` command receives an env var (which is a pytest marker) in both workflows [INTEGRATION_TEST_PYTEST_MARKERS](https://github.com/RasaHQ/rasa/pull/12187/files#diff-33b53b32f50aa98dc8f70274e80f6db68f4a807d1c75cafebc1370c2de296bd6R632) that is set to different values.
There are some tracker store tests that are marked with `pytest.mark.sequential` and only those will be run in the 2nd sequential workflow. The rest (not labelled as sequential) will be run with the non-sequential workflow.

Reply:
Yipeee 😄